ra-optimisation/ra-optimisation-api/src/test/resources/networkTest.uct

```diff
@@ -0,0 +1,20 @@
+##C 2007.05.01
```

Comment:
Files exemple_crac_xlsx.xlsx and networkTest.uct are not used in this module. Can you delete them?